cmd/lambda/populate_reset_queue/main.go

```diff
@@ -52,24 +52,19 @@ func Handler(cloudWatchEvent events.CloudWatchEvent) error {
 		Status: account.StatusNotReady.StatusPtr(),
 	}
 
-	for {
-		accounts, err := services.Config.AccountService().List(query)
-		if err != nil {
-			return err
-		}
-		for _, acct := range *accounts {
-			// Send Message
-			err = services.Config.AccountService().Reset(&acct)
-			if err != nil {
-				return err
+	return services.Config.AccountService().ListPages(query,
```

Comment:
For reference, I noticed that the AWS Golang SDK has started doing pagination with a callback like this:

for example:
https://docs.aws.amazon.com/sdk-for-go/api/service/athena/#Athena.ListNamedQueriesPagesWithContext